projectile.el

```diff
@@ -2702,6 +2702,12 @@ test/impl/other files as below:
                                   :test "pipenv run test"
                                   :test-prefix "test_"
                                   :test-suffix "_test")
+(projectile-register-project-type 'python-poetry '("poetry.lock")
+                                  :project-file "poetry.lock"
+                                  :compile "poetry build"
+                                  :test "poetry run python -m unittest discover"
+                                  :test-prefix "test_"
+                                  :test-suffix"_test")
```

Comment:
@jtpavlock I can't test this right now, but should this be

```suggestion
                                  :test-suffix "_test")
```

Reply:
this was fixed already by @bbatsov in 2eaf2c0e2d42457b55068d92659ef14195e12918

(fwiw, it worked without the space as well)